app/components/candidate_interface/application_complete_content_component.html.erb

```diff
@@ -32,16 +32,6 @@
     <%= t('application_complete.dashboard.application_not_sent', count: choice_count) %>
   </h2>
 <% elsif all_provider_decisions_made? %>
-  <h2 class="govuk-heading-m govuk-!-font-size-27">
-    <%= t('application_complete.dashboard.all_provider_decisions_made', count: choice_count) %>
-  </h2>
-
-  <% if any_offers? %>
-    <p class="govuk-body">
-      You have <%= decline_by_default_remaining_days %> days (until <%= decline_by_default_date %>) to respond to any offers. If you
-      do not respond, your applications will be automatically withdrawn.
-    </p>
-  <% end %>
 <% elsif any_offers? %>
```

Comment:
I think most of this file can be deleted now.

The page shouldn't ever show an h2 above the course choices any more, just the h1 ("Your applications"), then some paragraphs of text (or a call to action), and then the course choice boxes.

<img width="733" alt="Screenshot 2021-04-09 at 11 20 50" src="https://user-images.githubusercontent.com/30665/114166632-e6cd8380-9925-11eb-936a-1b6773bec3cc.png">

Reply:
thanks @frankieroberto I've removed the `h2`s but comparing the content with the prototype it looks like there are quite a few differences. Is the best approach to go through the prototype status by status and update the content from there?

Reply:
@stevehook yes, that's probably the easiest. Although do feel free to ask us (me or @Tomcowan1) if you have any questions, as it's possible that we've missed something.